R/derive_param_computed.R

```diff
@@ -252,7 +309,15 @@ derive_param_computed <- function(dataset = NULL,
   if (!is.null(set_values_to$PARAMCD) && !is.null(dataset)) {
     assert_param_does_not_exist(dataset, set_values_to$PARAMCD)
   }
-  analysis_value <- enexpr(analysis_value)
+  assert_logical_scalar(keep_nas)
+  if (!missing(analysis_value)) {
```

Comment:
```suggestion
  if (!missing(analysis_value) & !missing(analysis_var) {
```

Is there an edge case where this breaks if `analysis_var` is not included?

Reply:
Why should we add `& !missing(analysis_var)`?
It is always `TRUE` due to line 303:
```
analysis_var <- assert_symbol(enexpr(analysis_var))
```

Reply:
Ahh i missed that